src/main/resources/static/css/style.css

```
@@ -93,6 +113,68 @@ nav {
+    text-align: center;
+    position: relative; /* Viktigt för att kunna placera loggan bakom */
+    overflow: hidden;
+}
+
+/* Detta skapar loggan som ett eget lager bakom texten */
+.about-hero::before {
+    content: "";
+    position: absolute;
+    top: 0; left: 0; right: 0; bottom: 0;
+
+    background-image: url('/images/logo.png');
```

Stylelint (17.7.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)